openstates/oh/people.py

```diff
@@ -167,15 +177,16 @@ def scrape_page(self, chamber, term, url):
                 'sd{0:0{width}}@ohiosenate.gov'
             ).format(int(district), width=2)
 
-            leg = Legislator(term, chamber, district, full_name,
-                             party=party, url=homepage, photo_url=img)
+            leg = Person(name=full_name, district=district,
+                         party=party, primary_org=chamber,
+                         image=img)
 
-            leg.add_office('capitol', 'Capitol Office',
-                           address=office,
-                           phone=phone,
-                           email=email)
+            leg.add_contact_detail(type='address', value=office, note='Capitol Office')
+            leg.add_contact_detail(type='voice', value=phone, note='Capitol Office')
+            leg.add_contact_detail(type='email', value=email, note='Capitol Office')
 
-            self.scrape_homepage(leg, chamber, homepage, term)
+            self.scrape_homepage(leg, chamber, homepage)
```

Comment:
It looks like `scrape_homepage` is now a generator since it yields `Organization` objects, so we need to `yield from` it.